src/legacy/core_plugins/metrics/server/lib/vis_data/request_processors/table/date_histogram.js

```diff
@@ -23,17 +23,20 @@ import getIntervalAndTimefield from '../../get_interval_and_timefield';
 import getTimerange from '../../helpers/get_timerange';
 import { calculateAggRoot } from './calculate_agg_root';
 
-export default function dateHistogram(req, panel) {
+export default function dateHistogram(req, panel, esQueryConfig, indexPatternObject, capabilities) {
   return next => doc => {
     const { timeField, interval } = getIntervalAndTimefield(panel);
     const { bucketSize, intervalString } = getBucketSize(req, interval);
```

Comment:
It seems like `getBucketSize` has a third parameter `capabilities` which is important for rollup support - I didn't test but is has to get passed in here as well, right?

Reply:
@flash1293. Hm... not sure that it's so important for Table view because we use only last bucket to show data. But let's fix it

Reply:
Oh... no, it's really important to Rollup search. Thanks!